source/nodejs/adaptivecards/src/carousel.ts

```diff
@@ -199,13 +200,12 @@ export class Carousel extends Container {
         }
 
         const cardLevelContainer: HTMLElement = document.createElement("div");
-
-        const containerForAdorners: HTMLElement = document.createElement("div");
```

Comment:
this container should not be removed, this allows the arrows and pagination to be rendered on the sides and below the carousel correctly

Reply:
@almedina-ms, this is causing quite an issue. with this, rtl won't work correctly.